WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -0,0 +1,34 @@
+// Add params from the search fields to the bootstrap-table for on Ajax request.
+function queryParams(params) {
```

Comment:
Do we need this function defined at the global scope? If not, you should either move it into the `onPage` below, or wrap this whole file in an [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression).